src/test/java/com/celements/cleverreach/MailingConfigTest.java

```diff
@@ -59,7 +59,7 @@ public void testGetContentPlain() {
   @Test
   public void testGetContentHtmlCleanXml() {
     setUpMailingConf("<div>&nbsp;</div>\n");
-    assertEquals("<div>&#160;</div>\n", mailingConf.getContentHtmlCleanXml());
+    assertEquals("<div>&#160;</div>\n\n", mailingConf.getContentHtmlCleanXml());
```

Comment:
needed to fix this test with an additional `\n`. any idea why?

Reply:
I suppose due to the new "HTML5-JTidy" version, but no idea why that makes a second new line necessary for the test.